src/main/lrsql/admin/routes.clj

```diff
@@ -34,6 +34,13 @@
                                          ji/validate-jwt-account
                                          ai/create-admin)
      :route-name :lrsql.admin.account/create]
+    ;; Update account password
+    ["/admin/account/password"
+     :put (conj common-interceptors
```

Comment:
PUT is "replace" while POST is "create" right?

Reply:
Usually, though in practice different systems do different things.

Reply:
Yeah it seems lrsql follows convention (see create vs update API keys)